services/ui-src/src/components/reports/ReportPageIntro.tsx

```diff
@@ -14,57 +14,43 @@ export const ReportPageIntro = ({
   reportYear,
   ...props
 }: Props) => {
-  const { section, subsection, hint, info } = text;
+  const { section, subsection, info, introAccordion } = text;
 
-  if (section && (subsection || initiativeName)) {
+  const isSubsectionPage = section && (subsection || initiativeName);
+
+  const header = () => {
     return (
-      <Box sx={sx.introBox} {...props}>
+      <Heading as="h1" sx={sx.largeHeading}>
+        {subsection}
+      </Heading>
+    );
+  };
+
+  const headerSubsection = () => {
+    return (
+      <>
         <Heading as="h1" sx={sx.smallHeading}>
           {section}
         </Heading>
         <Heading as="h2" sx={sx.largeHeading}>
           {initiativeName ? initiativeName : subsection}
         </Heading>
-        {hint && <Box sx={sx.hintTextBox}>{parseCustomHtml(hint)}</Box>}
-        {accordion && !OverlayModalStepTypes.OBJECTIVE_PROGRESS && (
-          <InstructionsAccordion verbiage={accordion} />
-        )}
-        {OverlayModalStepTypes.OBJECTIVE_PROGRESS && text.dashboardTitle && (
-          <Heading as="h3" sx={sx.subTitle}>
-            {text.dashboardTitle}
-          </Heading>
-        )}
-        {info && <Box sx={sx.infoTextBox}>{parseCustomHtml(info)}</Box>}
-        {accordion && OverlayModalStepTypes.OBJECTIVE_PROGRESS && (
-          <InstructionsAccordion verbiage={accordion} />
-        )}
-        <ReportPeriod
-          text={text}
-          reportPeriod={reportPeriod}
-          reportYear={reportYear}
-        />
-      </Box>
+      </>
     );
-  }
+  };
 
   return (
     <Box sx={sx.introBox} {...props}>
-      <Heading as="h1" sx={sx.largeHeading}>
```

Comment:
The main difference between these two big blocks of JSX was the heading situation when on a subsection page vs a standard page. So I've extracted that out into a function and we have much less JSX to look at in this component.